src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    long draw();
```

Comment:
Return type should be `void`
```suggestion
    void draw();
```